src/curvefit/uncertainty/draws.py

```diff
@@ -97,7 +97,7 @@ def create_draws(self,
             model = model_prototype.clone()
             solver = solver_prototype.clone()
             solver.set_model_instance(model)
-            (current_group_data, data_specs) = data._get_df(group=group, copy=False, return_specs=True)
+            current_group_data, data_specs = data._get_df(group=group, copy=False, return_specs=True)
             solver.fit(data=(current_group_data, data_specs))
 
```

Comment:
@jizezhang is it how it's intended to be? I later use current_data_group as a pandas dataframe so I need to unpack these two.

Reply:
looks good!